resources/views/inputs/upload-image-single.blade.php

```diff
@@ -74,14 +74,14 @@ class="relative {{ $dimensions }}"
 >
     <div @class([
         'rounded-xl w-full h-full focus-within:border-theme-primary-500',
-        'p-2 border-2 border-dashed border-theme-primary-100' => ! $image,
+        'p-2 border-2 border-dashed border-theme-primary-100 dark:border-theme-secondary-800' => ! $image,
```

Comment:
It's not part of your PR, but when I look at the design in Figma, I think `p-2` should be `p-1.5`